app/views/purchases/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="container">
+<p style="color: green"><%= notice %></p>
+<div>
+<h1 class="header">Purchases</h1>
+<%= link_to "New purchase", new_category_purchase_path  ,class: "btn btn-info"%>
+</div>
+  <div id="purchases">
+        <%= render 'shared/purchase_table' do %>
+          <% @purchases.each do |purchase| %>
+            <%= render purchase %>
+          <% end %>
+        <% end %>
+      </div>
+
+      </div>
```

Comment:
- [ ] Please make sure at the top of the purchase(transaction) page the total amount for the category is displayed as requested in the project requirements.

![image](https://user-images.githubusercontent.com/83803414/201388676-e7c092d9-cc06-47ce-b14b-e2988ec4e64b.png)